sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -237,13 +249,55 @@ void deleteOnline(String partition, String documentId) {
 
     /**
      * Deletes the specified document from the cache.
+     *
      * @param pendingOperation Pending operation to delete.
      */
     void deletePendingOperation(PendingOperation pendingOperation) {
         deleteOnline(pendingOperation.getPartition(), pendingOperation.getDocumentId());
     }
 
-    List<PendingOperation> getPendingOperations(String partition) {
+    List<PendingOperation> getPendingOperations() {
+        return getPendingOperations(Constants.USER);
+    }
+
+    /**
+     * Validate partition name.
+     *
+     * @param partition name.
+     * @return true if the partition is supported, false otherwise.
+     */
+    static boolean isValidPartitionName(String partition) {
+        return READONLY.equals(partition) || USER.equals(partition);
+    }
+
+    void updatePendingOperation(PendingOperation operation) {
+
+        /*
+         * Update the document in cache (if expiration_time still valid otherwise, remove the document),
+         * clear the pending_operation column, update etag, download_time and document columns.
+         */
+        long now = Calendar.getInstance().getTimeInMillis();
```

Comment:
Any reason not to use `System.currentTimeMillis()` to avoid useless instance creations?